### PR TITLE
refactor: remove useless order getters

### PR DESCRIPTION
**Pull request by ghost**
Removed order getters that don't really do anything.

The ones I left either have:
* some nontrivial calculation
* some value in the form of pointing to which property you should use. like, orders have many types of prices, so it's good to have a single source of truth that points to that "one true" property to use

### CODE REVIEW

File: packages/theme/modules/checkout/getters/orderGetters.ts [hunk 1]
```diff
@@ -1,27 +1,17 @@
...
-export const getItemName = (item: OrderItemInterface): string => item?.product_name || '';
-
-export const getItemQty = (item: OrderItemInterface): number => item?.quantity_ordered || 0;
-
 export const getItemPrice = (item: OrderItemInterface): number => item?.product_sale_price?.value || 0;
 
-export const getFormattedPrice = (price: OrderItemInterface) => String(price);
-
-const getPagination = (orders: any): AgnosticPagination => ({
+const getPagination = (orders: CustomerOrders): AgnosticPagination => ({
```
Comment (sethidden) [inline packages/theme/modules/checkout/getters/orderGetters.ts:14]: removed any

File: packages/theme/modules/checkout/getters/orderGetters.ts [hunk 2]
```diff
@@ -1,27 +1,17 @@
 import { AgnosticPagination } from '~/composables/types';
-import { CustomerOrder, OrderItemInterface } from '~/modules/GraphQL/types';
+import { CustomerOrders, CustomerOrder, OrderItemInterface } from '~/modules/GraphQL/types';
 
 export const getDate = (order: CustomerOrder): string => new Date(order?.order_date?.replace(/ /g, 'T')).toLocaleDateString() || '';
 
-export const getId = (order: CustomerOrder): string => order?.number || '';
-
 export const getStatus = (order: CustomerOrder): string => order?.status || 'Failed';
```
Comment (bartoszherba) [inline packages/theme/modules/checkout/getters/orderGetters.ts:6]: Please, "failed" is not a good status, change it maybe to `unknown` or something that will mark it in a similar way.

File: packages/theme/modules/checkout/getters/orderGetters.ts [hunk 3]
```diff
@@ -1,27 +1,17 @@
...
 
 export const getPrice = (order: CustomerOrder): number | null => order?.total?.base_grand_total?.value || 0;
```
Comment (bartoszherba) [inline packages/theme/modules/checkout/getters/orderGetters.ts:8]:
```suggestion
export const getPrice = (order: CustomerOrder): number => order?.total?.base_grand_total?.value ?? 0;
```

File: packages/theme/modules/checkout/getters/orderGetters.ts [hunk 4]
```diff
@@ -1,27 +1,17 @@
 import { AgnosticPagination } from '~/composables/types';
-import { CustomerOrder, OrderItemInterface } from '~/modules/GraphQL/types';
+import { CustomerOrders, CustomerOrder, OrderItemInterface } from '~/modules/GraphQL/types';
```
Comment (bartoszherba) [inline packages/theme/modules/checkout/getters/orderGetters.ts:2]:
```suggestion
import type { CustomerOrders, CustomerOrder, OrderItemInterface } from '~/modules/GraphQL/types';
```

File: packages/theme/modules/checkout/getters/orderGetters.ts [hunk 5]
```diff
@@ -1,27 +1,17 @@
...
 
 export const getDate = (order: CustomerOrder): string => new Date(order?.order_date?.replace(/ /g, 'T')).toLocaleDateString() || '';
```
Comment (bartoszherba) [inline packages/theme/modules/checkout/getters/orderGetters.ts:4]:
```suggestion
export const getDate = (order: CustomerOrder): string => new Date(order?.order_date?.replace(/ /g, 'T')).toLocaleDateString();
```

**Review by bartoszherba — CHANGES_REQUESTED** [review]

File: packages/theme/modules/checkout/getters/orderGetters.ts (hunk 2 above)
Comment (sethidden) [reply packages/theme/modules/checkout/getters/orderGetters.ts]: Removed geStatus entirely 8bb720ab4fd6afea392cd0c02b37c8a02a1762c8

File: packages/theme/modules/checkout/getters/orderGetters.ts (hunk 3 above)
Comment (sethidden) [reply packages/theme/modules/checkout/getters/orderGetters.ts]: 8bb720ab4fd6afea392cd0c02b37c8a02a1762c8

File: packages/theme/modules/checkout/getters/orderGetters.ts (hunk 4 above)
Comment (sethidden) [reply packages/theme/modules/checkout/getters/orderGetters.ts]: 8bb720ab4fd6afea392cd0c02b37c8a02a1762c8

File: packages/theme/modules/checkout/getters/orderGetters.ts (hunk 5 above)
Comment (sethidden) [reply packages/theme/modules/checkout/getters/orderGetters.ts]: 8bb720ab4fd6afea392cd0c02b37c8a02a1762c8

**Review by bartoszherba — APPROVED** [review]

**Pull request merged**